script_editor/app/assets/stylesheets/plays.scss

```diff
@@ -392,11 +392,11 @@ milestone{ display: block; }
 
 // I don't know why this doesn't work for Act and Scene headers... 
```

Comment:
We should probably get rid of this comment if it doesn't apply to our version because I thought it was working for act and scene headers?